codes/quantum/qubits/two_dimensional_hyperbolic_surface.yml

```diff
@@ -0,0 +1,106 @@
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: '\( k/n = (1-2/r - 2/s) + 2/n \) which approaches a constant value as the number of physical qubits grows. The distance of the code scales logarithmically with \( n \) and the weight of the stabilizers is \( r \) for \(r\)-checks and \(s\) for \(X\)-checks, ( Recall \( {r,s} \) are the tiling parameters for the hjyperbolic surface)'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders:
+    - 'As with other surface codes, after the surface is tessellated, the physical qubits sit on the edges of the tiles and the logical qubits sit at the center of the unit.'
+    - 'For each family of codes defined by a given tessellation (ie, {r,s}) the encoding rate decreases with $n$, thus a [[n,k,d]] code derived from a {r,s} tiling will have the rate $k/n = 1 - 2/r - 2/s + 2/n$'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+
+
+  decoders:
+    - 'Many decoders developed fro the toric code (surface code) can also be used for hyperbolic surface codes. For example, minimum weight-perfect matching \cite{arXiv:1506.04029} and union find decoder \cite{1709.06218} are both valid decoders for two-dimensional hyperbolic surface codes.'
+    - 'Due to the symmetries of hyperbolic surface codes, optimal measurement schedules of the stabilizers can be found \cite{arXiv:2010.09626}.'
+
+  # fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  threshold:
+    - 'Since the distance is logarithmic, the threshold exists for a given tiling under minimum weight decoding \cite{doi:10.1103/PhysRevA.87.020304}.'
+    - 'For a {5,4} tiling, the threshold is between 1% - 5% (compared to the toric code 10.3%)'
+    - 'For larger tilings, the lower bound on the distance decreases suggesting the threshold will also decrease.'
+
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'This family of codes is described on a closed 2D surface, however, in a physical implementaion the surface would have open boundary conditions. Creating explicit boundaries reduces the efficiency of the code, reducing the code to a constant distance. '
```

Comment:
This can be tacked onto description of code. On a similar note, I tack on a qualifier about non-normalizability for infinite-dim code states for oscillators.

Reply:
Added the closed vs open surface sentence to the description. Still need to look into the oscillator point

Reply:
We disuccsed this - not applicable to this code